### ISSUE TITLE
Generate only imports which will be used

### ISSUE THREAD

**Issue by capaj** [MichalLytek/typegraphql-prisma#161]
**Is your feature request related to a problem? Please describe.**
Currently generated files often have imports that are not used anywhere
![image](https://user-images.githubusercontent.com/1305378/129638919-859d58cd-e4b5-4da0-8d40-83404e1cb032.png)

**Describe the solution you'd like**
Unused import would be stripped/never even generated

**Describe alternatives you've considered**
stripping it myself using eslint

**Additional context**

**Comment by MichalLytek** [comment]
I tried to use `ts-morph` optimize functionality but it's super super slow that it's unusable 😞

**Comment by JClackett** [comment]
Yeah if there was a way to do this, that would be great! currently generating the source files is not an option as it then fails all eslint/ts checks.

**Comment by MichalLytek** [comment]
@JClackett You should just ignore that generated source files folder in eslint or put a dedicated tsconfig for that folder to supress the unused stuff errors.

`ts-morph` optimize functionality is unusable and doing such import optimization manually in the code is a tedious task - there are plenty of issues with more priority, so unless someone make a PR with that, it won't happen soon 😞

**Comment by JClackett** [comment]
Yeah for sure this isn't high priority, so all good! :)
